src/esssans/i_of_q.py

```diff
@@ -171,7 +172,7 @@ def merge_spectra(
     data: CleanQ[RunType, IofQPart],
     q_bins: QBins,
     wavelength_bins: WavelengthBins,
```

Comment:
@nvaytet Is this unused now? Looks like it.

As I am working on a follow-up change, I can just do it there, if you confirm that it should indeed be removed.